qa/backwards/shared/src/test/java/org/elasticsearch/bwcompat/BasicBackwardsCompatibilityIT.java

```diff
@@ -713,4 +715,55 @@ private static String indexOrAlias() {
     private void createIndexWithAlias() {
         assertAcked(prepareCreate("test").addAlias(new Alias("alias")));
     }
+
+    public void testSimpleFunctionScoreParsingWorks() throws IOException, ExecutionException, InterruptedException {
```

Comment:
I thought it would be faster to have the test in this class which has already started a cluster instead of starting an own cluster just for it. Before it needed that because of the node settings but now not anymore. But I'll keep it in mind next time!